### PR TITLE
Added documentation for DbalSessionHandler as there is none

### PR DESCRIPTION
**Pull request by ghost**
I've written this documentation for the DbalSessionHandler as there was none, at least, none that was specifically about this class (even though one exists for the PdoSessionHandler).

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -13,6 +13,17 @@ documentation.
     Do you also want to participate in the Symfony Documentation? Take a look
     at the ":doc:`/contributing/documentation/overview`" article.
 
+November, 2016
```
Comment (wouterj) [inline changelog.rst]: The changelogs are automatically generated based on the merge commits. Please remove the changes in this file.

Comment (hipio) [inline changelog.rst]: My apologies. I'll remove the change log changes!

**Comment by javiereguiluz** [comment]
DbalSessionHandler is probably [going to be deprecated](https://github.com/symfony/symfony/issues/20501). In that case, should we document it?

File: changelog.rst [hunk 2]
```diff
@@ -13,78 +13,6 @@ documentation.
     Do you also want to participate in the Symfony Documentation? Take a look
     at the ":doc:`/contributing/documentation/overview`" article.
 
-October, 2016
```
Comment (HeahDude) [inline changelog.rst]: Those changes should be reverted.

**Review by HeahDude — APPROVED** [review]
This PR seems nice and ready to me (apart a minor comment.

So I'm in favor of merging it. Thanks!

**Comment by hipio** [comment]
@HeahDude If this documentation is still wanted (seeing as DbalSessionHandler is to be deprecated), I'll revert the changelog.rst tonight.

**Comment by fabpot** [comment]
Closing this issue as the code is being deprecated.

**Pull request closed without merging**